typescript/packages/well-log-viewer/src/WellLogViewer.tsx

```diff
@@ -289,6 +294,7 @@ export default class WellLogViewer extends Component<
                     <WellLogViewWithScroller
                         /* just copy all props without primaryAxis */
                         welllog={this.props.welllog}
+                        wellLogSets={this.props.wellLogSets}
```

Comment:
wellLogSets={this.props.wellLogSets ?? this.props.welllog}

and remove line above

Reply:
Fine for me.
But you need to take care of the defaults for onTrackMouseEvent and layout